tests/test_plot.py

```diff
@@ -29,24 +29,34 @@ def test_show_raster():
             show((src, 1))
             fig = plt.gcf()
             plt.close(fig)
+
         except ImportError:
             pass
 
-        with rasterio.open('tests/data/RGB.byte.tif') as src:
-            try:
-                show(src)
-                fig = plt.gcf()
-                plt.close(fig)
-            except ImportError:
-                pass
+    with rasterio.open('tests/data/RGB.byte.tif') as src:
+        try:
+            show((src, (1, 2, 3)))
+            fig = plt.gcf()
+            plt.close(fig)
+
+        except ImportError:
```

Comment:
The `try/except ImportError` dance we did when we originally added this test could be eliminated and simplified by using pytest's [importorskip](https://docs.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency). Would you be willing to try that out in this PR?